spec/abci/abci++_methods.md

```diff
@@ -93,26 +93,26 @@ title: Methods
-    | info      | string                | Additional information. **May be non-deterministic.**                                                                                                                                                              | 4            |
-    | index     | int64                 | The index of the key in the tree.                                                                                                                                                                                  | 5            |
-    | key       | bytes                 | The key of the matching data.                                                                                                                                                                                      | 6            |
-    | value     | bytes                 | The value of the matching data.                                                                                                                                                                                    | 7            |
-    | proof_ops | [ProofOps](#proofops) | Serialized proof for the value data, if requested, to be verified against the `app_hash` for the given Height.                                                                                                     | 8            |
-    | height    | int64                 | The block height from which data was derived. Note that this is the height of the block containing the application's Merkle root hash, which represents the state as it was after committing the block at Height-1 | 9            |
-    | codespace | string                | Namespace for the `code`.                                                                                                                                                                                          | 10           |
+    | Name      | Type                  | Description                                                                                                                                                                                                        | Field Number | Deterministic |
+    |-----------|-----------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|--------------|---------------|
+    | code      | uint32                | Response code.                                                                                                                                                                                                     | 1            | No            |
```

Comment:
For `Query`, rather than "No", I'd say it's an "N/A", and that ABCI method is not involved in SMR, it's rather a local query